src/dmd/mars.d

```diff
@@ -266,7 +266,8 @@ private int tryMain(size_t argc, const(char)** argv)
     sections.push(envsection.ptr);
     parseConfFile(&environment, global.inifilename, inifilepath, inifile.len, inifile.buffer, &sections);
     getenv_setargv(readFromEnv(&environment, "DFLAGS"), &arguments);
-    updateRealEnvironment(&environment);
+    // TODO: not sure why dmd is doing this
+    //updateRealEnvironment(&environment);
```

Comment:
Because it has the "great concept" of replacing `DFLAGS` completely with the content of the INI file.
See also: https://github.com/dlang/tools/pull/343